kernelci/build.py

```diff
@@ -87,6 +87,7 @@ def get_last_commit(config, storage):
         file_name=_get_last_commit_file_name(config))
     last_commit_resp = requests.get(last_commit_url)
     if last_commit_resp.status_code != 200:
+        print(f'get_last_commit() HTTP error: {last_commit_resp.status_code}')
```

Comment:
```suggestion
        print(f'Failed to retrieve the last commit. HTTP error code: {last_commit_resp.status_code}')
```

Reply:
Updated, i still need function name as we have several places in code where we retrieve last commit, also in some cases HTTP code might be not error, but just unexpected(not 200) HTTP code.